cijoe/scripts/gha_prepare_python.py

```diff
@@ -42,8 +42,20 @@ def main(args, cijoe, step):
         log.error(f"invalid step({step})")
         return errno.EINVAL
 
+    # TODO: because pipx / userpath only adds PATH to files read by
+    # login-shells, then we cannot use 'pipx ensurepath' anymore.
+    # Issue has been reported here:
+    #
+    # * https://github.com/pypa/pipx/pull/1021
+    # * https://github.com/ofek/userpath/issues/50
+    #
```

Comment:
I think this should be an issue on github instead of a todo in the code

Reply:
Yeah, changed to a NOTE instead. As the code needed a comment...